kimchi/src/circuits/polynomials/xor.rs

```diff
@@ -6,7 +6,7 @@ use crate::{
     circuits::{
         argument::{Argument, ArgumentEnv, ArgumentType},
         constraints::ConstraintSystem,
```

Comment:
the file and module are called xor because i want to be general at the library level, but then the gate gives further information about how it works. The only two things I can recall are either `xor.rs` and `Xor16`, I don't think there's any xor16 anywhere

Reply:
Sounds good